hypothesis-python/tests/nocover/test_floating.py

```diff
@@ -106,7 +107,9 @@ def test_is_in_exact_int_range(x):
 # This is essentially a function of how the python interpreter
 # was compiled.
 # Everything is terrible
-if math.ldexp(0.25, -1022) > 0:
+# (Also for some reason subnormals aren't working on Python 3.8, which was
+# being masked by a test bug.)
+if math.ldexp(0.25, -1022) > 0 and sys.version_info[:2] != (3, 8):
```

Comment:
https://github.com/HypothesisWorks/hypothesis/blob/0141a73abd0facfec1c931643b2ffde903646dbc/hypothesis-python/scripts/basic-test.sh#L57-L61

I suspect that this doesn't work on *any* Python version, since we're only running the `nocover` tests on Python 3.8 😱

It's fine if you want to slap `@pytest.mark.xfail` on them and open an issue instead of fixing it in this PR; that's a strict improvement on the status quo and worth merging as such.

Reply:
Oh you're right; I though I had tested them locally on 3.9 but I tried them just now and they fail. 🤷‍♂️ 

Yeah, I'll just mark them `xfail` for now.